src/System.Private.CoreLib/shared/Interop/Unix/System.Globalization.Native/Interop.Collation.cs

```diff
@@ -12,19 +12,20 @@ internal static partial class Interop
     internal static partial class Globalization
     {
         [DllImport(Libraries.GlobalizationNative, CharSet = CharSet.Unicode, EntryPoint = "GlobalizationNative_GetSortHandle")]
-        internal static extern unsafe ResultCode GetSortHandle(byte[] localeName, out SafeSortHandle sortHandle);
+        internal static extern unsafe ResultCode GetSortHandle(byte[] localeName, out CriticalSortHandle sortHandle);
 
+        [SuppressUnmanagedCodeSecurity]
```

Comment:
Why is this needed?

Reply:
That shouldn't be needed at all. In fact, I believe we entirely ignore that attribute in .NET Core.

Reply:
@stephentoub I took this advice from this CriticalHandle comment:

https://github.com/dotnet/coreclr/blob/9773db1e7b1acb3ec75c9cc0e36bd62dcbacd6d5/src/System.Private.CoreLib/shared/System/Runtime/InteropServices/CriticalHandle.cs#L83-L87

Reply:
I see, thanks. That comment appears to be stale and isn't relevant to .NET Core.

Reply:
Would be good to fix the comment.